triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -17,24 +17,26 @@ import com.daml.lf.speedy.{ArrayList, SValue}
 import com.daml.lf.speedy.SValue._
 import com.daml.lf.value.Value.ContractId
 import com.daml.ledger.api.v1.commands.{
-  Command,
   CreateAndExerciseCommand,
   CreateCommand,
   ExerciseByKeyCommand,
   ExerciseCommand,
+  Command => ApiCommand,
```

Comment:
As we will be referring to both Speedy and Ledger `Command` types, aliasing the type name